pkgs/applications/networking/instant-messengers/telegram/tdesktop/tg_owt.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, cmake, ninja, yasm
+, pkg-config, libjpeg, openssl, libopus, ffmpeg, alsaLib, libpulseaudio
+}:
+
+let
+  rev = "c73a4718cbff7048373a63db32068482e5fd11ef";
+  sha256 = "0nr20mvvmmg8ii8f2rljd7iv2szplcfjn40rpy6llkmf705mwr1k";
+
+in stdenv.mkDerivation {
+  pname = "tg_owt";
+  version = "git-${rev}";
+
+  src = fetchFromGitHub {
+    owner = "desktop-app";
+    repo = "tg_owt";
+    inherit rev sha256;
+  };
+
+  patches = [ ./tg_owt-install.patch ];
+
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace "@@source@@" "$src"
+  '';
+
+  nativeBuildInputs = [ cmake ninja yasm ];
+
+  buildInputs = [ pkg-config libjpeg openssl libopus ffmpeg alsaLib libpulseaudio ];
+
+  postInstall = ''
+    ln -s $src/src $out/include
```

Comment:
Could we copy them to $dev output?

Reply:
Btw, I think that symlink is not going to work, since it references the directory in src. Instead it should be cp -r

Reply:
@jtojnar 
> Could we copy them to $dev output?

This derivation itself is a static library. It can only be used for development.
I think there's no need to separate $out and $dev.

Reply:
Should be fine in this case.